css-two-column-layout/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header class="header-center">
+      <h1>SCALE, SPEED, AGILITY</h1>
+      <p>We've dedicated thousands of man hours to making healthcare instiutions
+        in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.
+      </p>
+    </header>
+
+    <div class="info-items">
+      <section class="row">
+        <img src="images/software.svg" alt="Picture of Africa"  class="info-images"/>
+        <div class="info-text">
+          <h3>INTUITIVE, SMART SOFTWARE</h3>
+          <p>We build products that transform how healthcare facilities run across Africa -
+            making them radically more efficient operationally and financially, a complete
+            turnaround.
+          </p>
+        </div>
+
+        <div class="column-one-tenth"></div>
```

Comment:
😱 😱 😱 😱 😱 
You should _**never**_ use empty elements to create spacing. This is a job for `margin` or `padding`